docker-compose.yaml

```diff
@@ -24,14 +24,10 @@ services:
     container_name: airbyte-bootloader
     environment:
       - AIRBYTE_VERSION=${VERSION}
-      - CONFIG_DATABASE_PASSWORD=${CONFIG_DATABASE_PASSWORD:-}
```

Comment:
why are we removing these?

Reply:
They are no longer used.  We reference the `DATABASE_X` env vars only.  We could reintroduce the separate ones if necessary.

Reply:
ah. if these are no longer used, we should also remove them from the EnvConfigs file.

although, maybe a good idea to wait on that since we haven't yet figure out how we want to track all configuration information globally. i.e. a master list of configuration that our micronaut application yamls accept.

Reply:
no need to action now. thinking out loud with you

Reply:
@davinchia Agreed.  Once everything is converted to Micronaut, there will be a pass to remove EnvConfigs entirely and clean up the environment variables.

Reply:
and I hope consolidate into one master list for reference :)